.circleci/config.yml

```diff
@@ -55,13 +55,7 @@ commands:
             - yarn-deps-v3-
       - run:
           name: Install dependencies
-          # upgrade Yarn before installing dependencies, if needed
-          command: |
-              if ! yarn check; then
-                curl -o- -L https://yarnpkg.com/install.sh | PROFILE=$BASH_ENV bash
-                source $BASH_ENV
-                yarn
-              fi
+          command: yarn
```

Comment:
Should we use `yarn install --frozen-lockfile` instead, as mentioned in https://github.com/yarnpkg/yarn/issues/6427#issuecomment-435831173 ?  I admit I don't really understand the implications either way, but it seems this would force it to respect `yarn.lock` and not drift?